ddtrace/ext/tags.go

```diff
@@ -107,3 +107,9 @@ const (
 	// SpanKind defines the kind of span based on Otel requirements (client, server, producer, consumer).
 	SpanKind = "span.kind"
 )
+
+// Messaging tags.
+const (
```

Comment:
Are we making a separate const block for every type of tag? Is this the best way to go or is it redundant?